README.rst

```diff
@@ -18,12 +18,12 @@
 dwave-system
 ============
 
-`dwave-system` is a basic API for easily incorporating the D-Wave system as a sampler in
-the `D-Wave Ocean <http://dw-docs.readthedocs.io/en/latest/overview/stack.html#stack>`_ software
-stack. It includes DWaveSampler, a :class:`dimod.Sampler`
-that accepts and passes system parameters such as system identification and authentication
-down the stack. It also includes several useful composites---layers of pre- and post-processing---that
-can be used with DWaveSampler to handle minor-embedding, optimize chain strength, etc.
+`dwave-system` is a basic API for easily incorporating the D-Wave system as a sampler in the
+`D-Wave Ocean software stack <https://docs.ocean.dwavesys.com/en/latest/overview/stack.html>`_\ .
+It includes DWaveSampler, a :class:`dimod.Sampler` that accepts and passes system parameters
```

Comment:
I don't like using the `:class:` type definitions in the README because they are only rendered in sphinx but not in the github landing page.